local-setup/jupyter/dataloader/crs_loader.py

```
@@ -367,75 +559,305 @@ def _bootstrap_tenant_root(self, target_root: str, source_tenant: str = "pg") ->
+    def _copy_stateinfo_direct(self, source_tenant: str, target_root: str):
+        """Copy StateInfo from source tenant to target root via direct DB operations"""
+        try:
+            import psycopg2
+            
+            # Database connection
+            conn = psycopg2.connect(
+                host="postgres-db",
+                port="5432", 
+                database="egov",
+                user="egov",
+                password="egov123"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)